.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,14 @@
+name: Update Github Pages
+on:
+  push: { branches: master }
```

Comment:
change this up to something like 
```
on:
  workflow_dispatch:
  pull_request:
```
to get this action to run manually (workflow_dispatch) and on PR. Remove these extra triggers once it's working and leave the push to master.

Reply:
OK, can do 👌